themes/wdfn_theme/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "wdfn_theme",
+  "version": "1.0.0",
+  "description": "USGS Water Data For The Nation Blog",
+  "module": "src/js/main.js",
+  "sass": "src/styles/main.scss",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "clean": "rm -rf static && mkdirp static",
+    "build": "npm run clean && run-p build:* && npm run hash-assets",
+    "build:css": "mkdirp static/css && node-sass --include-path node_modules/wdfn-viz/src/stylesheets --include-path node_modules/@fortawesome/fontawesome-free/scss src/styles/main.scss static/css/main.css && postcss static/css/main.css -o static/css/main.css",
+    "build:fonts": "mkdirp static/fonts && cp src/fonts/* static/fonts && cp -r node_modules/wdfn-viz/dist/fonts/* static/fonts && cp node_modules/@fortawesome/fontawesome-free/webfonts/* static/fonts",
+    "build:images": "mkdirp static/img && cp -r src/images/* static/img && cp -r node_modules/wdfn-viz/dist/img/* static/img",
+    "build:js": "mkdirp static/scripts && rollup -c --environment NODE_ENV:production",
+    "hash-assets": "gulp --gulpfile ./gulpfile.js revall",
+    "watch": "run-p watch:*",
+    "watch:css": "nodemon -w src/styles -e scss -x \"npm run build:css\"",
+    "watch:fonts": "npm run build:fonts",
+    "watch:images": "npm run build:images",
+    "watch:js": "rollup -c --watch"
+  },
+  "engines": {
+    "node": "8.12.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/usgs/wdfn-updates.git"
+  },
+  "keywords": [
+    "blog",
+    "water",
+    "hugo",
+    "usgs",
+    "wdfn"
+  ],
+  "author": "USGS",
+  "license": "CC0-1.0",
+  "bugs": {
+    "url": "https://github.com/usgs/wdfn-updates/issues"
+  },
+  "homepage": "https://github.com/usgs/wdfn-updates#readme",
+  "dependencies": {
+    "@fortawesome/fontawesome-free": "^5.3.1",
+    "wdfn-viz": "^0.1.9"
+  },
+  "devDependencies": {
+    "autoprefixer": "^9.1.5",
+    "css-mqpacker": "^7.0.0",
+    "cssnano": "^4.1.4",
+    "gulp": "^3.9.1",
+    "gulp-rev-all": "^1.0.0",
+    "jquery": "^2.2.4",
+    "mkdirp": "^0.5.1",
+    "node-sass": "^4.10.0",
+    "nodemon": "^1.18.4",
+    "npm-run-all": "^4.1.3",
+    "postcss-cli": "^6.0.0",
+    "postcss-flexbugs-fixes": "^4.1.0",
+    "rollup": "^0.66.2",
+    "rollup-plugin-alias": "^1.4.0",
+    "rollup-plugin-buble": "^0.19.2",
+    "rollup-plugin-commonjs": "^9.1.8",
+    "rollup-plugin-node-resolve": "^3.4.0",
+    "rollup-plugin-uglify": "^6.0.0",
+    "uglify-js": "^3.4.9"
+  }
+}
```

Comment:
has anyone looked at the version updates for these dependencies? (should we?)

Reply:
Yeah, I was going to look into getting Dependabot running and scanning the dependencies. I'm sure some of them could use updating.